macroslib/tests/test_expectations.rs

```diff
@@ -574,8 +574,10 @@ fn collect_code_in_dir(dir_with_code: &Path, exts: &[&str]) -> Result<String, Er
                 .iter()
                 .any(|ext| path.path().to_str().map_or(false, |x| x.ends_with(ext)))
         {
+            code.push_str(format!("<<< generated file: {} >>>\n", path.file_name().to_string_lossy()).as_str());
```

Comment:
You can write just {:?} to print `path.file_name()` after all this is test, there is no need for super robust solution, robust is enough

Reply:
Modified as indicated.

Except this, are you ok with the marker "design" ?
Anything else to suggest ?

Reply:
`{:?}` changes the output a little bit --> test cases failure.
Test case fixed with this change.